understanding/20/sensory-characteristics.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Sensory Characteristics</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Instructions are understandable by more people</dd>
+		<dt>Author task</dt><dd>Don't use descriptions reliant on using our five senses</dd>
```

Comment:
> Don't use descriptions reliant on using our five senses

This seems confusing and maybe even wrong. 
Author task, proposed lternative: "Don't use instructions or descriptions that rely on just one sense"

Reply:
I'll double check SC language, thanks.

Reply:
So, it's not just that instructions don't rely on one sense; it's that they don't rely on any sense. In other words, you need to name things. I wonder if we can go on a slightly different tack for this one. How about:
> Describe controls by name, not just by appearance or location

Although the SC is about 'understanding and operating content', in practical usage, instructions are almost always going to be about operation, so I think making it about controls makes the concept immediately more understandable.  As well, the only existing technique is "Providing textual identification of items..."
We could make it content, but every example in the technique is about naming controls
> Describe content by name, not just by appearance or location